ci_framework/roles/tempest/tasks/configure-tempest.yml

```diff
@@ -0,0 +1,32 @@
+- name: Get keystone secret name
+  register: keystone_secret_name
+  environment:
+    PATH: "{{ cifmw_path }}"
+  ansible.builtin.shell:
+    cmd: oc get keystoneapi keystone -o json | jq -r .spec.secret
+
+- name: Get keystone passwd select
+  register: keystone_passwd_select
+  environment:
+    PATH: "{{ cifmw_path }}"
+  ansible.builtin.shell:
+    cmd: oc get keystoneapi keystone -o json |jq -r .spec.passwordSelectors.admin
```

Comment:
Idea of module sounds good - but i think esp we should have on dedicat role for preparing the 'clouds.yaml' info.

As both os-net-config and tempest - and anything which wants to speak with OpenStack API - will need this info - so i was thinking about having it extracted to separate place which all can call/depend on - and it would obtain and generate the info file in -data/ (and populate the var) - if it does not already exist (later if needed 'force-regenerate' or support for different 'clouds'/'users' could be added into it if there would be need for such situations - but generally it could stay just in one place and be well maintained bit.